src/App.js

```diff
@@ -1,29 +1,40 @@
-import { Link, Route, Routes } from 'react-router-dom';
+import { NavLink, Route, Routes } from 'react-router-dom';
 import React from 'react';
 import Books from './components/Books';
 import Categories from './components/Categories';
 
 const App = () => (
-  <div>
+  <>
     <header>
```

Comment:
- [ ] Greate work implementing the design, however, you should duplicate the design. as for now, your nav bar doesn't have the profile icon, kindly implement it

![Screenshot_20220610_061654](https://user-images.githubusercontent.com/25227022/172995340-2f341371-98ef-41b1-9806-ed0401150459.png)

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220610_061251](https://user-images.githubusercontent.com/25227022/172995065-7142e612-2a23-44bc-891a-1dc1cd6cae17.png)|![Screenshot_20220610_061639](https://user-images.githubusercontent.com/25227022/172995273-1e43952f-269e-4804-887b-502db0c21786.png)|